deployment/data-streams/changeset/verifier/v0_5_0/call_set_config_test.go

```diff
@@ -0,0 +1,91 @@
+package v0_5_0
+
+import (
+	"encoding/hex"
+	"math/big"
+	"strings"
+	"testing"
+
+	"github.com/ethereum/go-ethereum"
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/stretchr/testify/require"
+
+	"github.com/smartcontractkit/chainlink-testing-framework/lib/utils/testcontext"
+	commonChangesets "github.com/smartcontractkit/chainlink/deployment/common/changeset"
+	"github.com/smartcontractkit/chainlink/deployment/data-streams/changeset/testutil"
+	verifier "github.com/smartcontractkit/chainlink/v2/core/gethwrappers/llo-feeds/generated/verifier_v0_5_0"
+)
+
+func TestCallSetConfig(t *testing.T) {
+	e := testutil.NewMemoryEnv(t, true)
+	ctx := testcontext.Get(t)
+	e, _, verifierAddr := DeployVerifierProxyAndVerifier(t, e)
+
+	var configDigest [32]byte
+	cdBytes, _ := hex.DecodeString("1234567890abcdef1234567890abcdef")
+	copy(configDigest[:], cdBytes)
+
+	signers := []common.Address{
+		common.HexToAddress("0x1111111111111111111111111111111111111111"),
+		common.HexToAddress("0x2222222222222222222222222222222222222222"),
+		common.HexToAddress("0x3333333333333333333333333333333333333333"),
+		common.HexToAddress("0x4444444444444444444444444444444444444444"),
+	}
+	f := uint8(1)
+
+	setConfigPayload := SetConfig{
+		VerifierAddress:            verifierAddr,
+		ConfigDigest:               configDigest,
+		Signers:                    signers,
+		F:                          f,
+		RecipientAddressesAndProps: []verifier.CommonAddressAndWeight{},
+	}
+
+	callCfg := SetConfigConfig{
+		ConfigsByChain: map[uint64][]SetConfig{
+			testutil.TestChain.Selector: {setConfigPayload},
+		},
+		MCMSConfig: nil,
+	}
+
+	e, err := commonChangesets.Apply(t, e, nil,
+		commonChangesets.Configure(
+			SetConfigChangeset,
+			callCfg,
+		),
+	)
+	require.NoError(t, err)
+
+	verifierABI, err := abi.JSON(strings.NewReader(verifier.VerifierMetaData.ABI))
+	require.NoError(t, err)
+
+	filter := ethereum.FilterQuery{
+		FromBlock: big.NewInt(0),
+		ToBlock:   nil,
+		Addresses: []common.Address{verifierAddr},
+		Topics:    [][]common.Hash{{verifierABI.Events["ConfigSet"].ID}},
+	}
+	logs, err := e.Chains[testutil.TestChain.Selector].Client.FilterLogs(ctx, filter)
+	require.NoError(t, err)
+	require.NotEmpty(t, logs, "Expected at least one ConfigSet event")
+
+	for _, lg := range logs {
```

Comment:
I think it'd be possibly easier to just use this generated log filter method https://github.com/smartcontractkit/chainlink/blob/d2eb42dd6e9244ebeed8401f30d30ca6e8df8b59/core/gethwrappers/llo-feeds/generated/verifier_v0_5_0/verifier_v0_5_0.go#L672-L672